### DISCUSSION TITLE
Incorporating GrahpQL

### DISCUSSION THREAD

**Discussion by mdaymard1** [skiptools/skip#302] [Q&A]
Hi,

I'm brand new to Skip and have read a lot of the docs and played with the TravelPosters, which is similar to what I would like to accomplish in terms of sharing the models and services with Android. But I'm not sure if I'm understanding the process correctly. The challenge I have is that my app needs to use the Apollo GraphQL SDK, and since I'm assuming that can't be transpiled to Android, I need to share my models and graphql services from iOS. But I also want to use the transpiler for my SwiftUI views for Android so that I only have to write them once. Is this possible? The TravelPosters app shares the model, but it looks like the Android view is using standard compose and not the SwiftUI Android libraries, so it appears that in this app, the views are maintained separately.

Any help would be greatly appreciated.

Thanks,
Mike

**Comment by flymedllva** [comment]
@mdaymard1 hi i see you closed the discussion, did you manage to get [apollo](https://github.com/apollographql/apollo-ios) running? @marcprux I think support for such libraries is very important for platform development. Any thoughts on the possibility of running [this library](https://github.com/apollographql/apollo-ios)?
